lib/SILOptimizer/Transforms/OwnershipModelEliminator.cpp

```diff
@@ -382,6 +382,9 @@ struct OwnershipModelEliminator : SILModuleTransform {
       if (SkipTransparent && F.isTransparent())
         continue;
 
+      // Verify here to make sure ownership is correct before we strip.
+      F.verify();
```

Comment:
When this is recommitted, I think it should only run ownership verification (with respect to other verification, there's nothing special about this point).